### PR TITLE
first draft for the required number of RSEs

### PR DESCRIPTION
**Pull request by CaptainSifff**
So, as promised I created  numbers on the number of required RSEs.
Feel free to discuss!

### CODE REVIEW

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -47,6 +47,18 @@ _[Overview of paper sections to add]_
 
 The challenges of understanding the current state of skills within the research software community and related areas, as well as identifying required competencies, developing training pathways and providing training materials are areas that are being looked at and addressed by various groups and projects. In this section, we highlight some of these other projects and activities.
 
+## Number of required RSE graduates
+In order to set up the argument for dedicated RSE education we need to estimate the number of required graduates.
+We start from the number of researchers in Germany. The OECD Data [OECD2023] reports a number of 667,394 researchers in 2019.
+In public research alone we have better numbers and there have been 71733 researcher in 2020.
```
Comment (jpthiele) [inline paper_teaching-learning-RSE.md:53]: What exactly is meant by public researches, if OECD lists government researchers? University only? University + Helmholtz, RKI, DLR, etc.?

File: paper_teaching-learning-RSE.md [hunk 2]
```diff
@@ -47,6 +47,18 @@ _[Overview of paper sections to add]_
 
 The challenges of understanding the current state of skills within the research software community and related areas, as well as identifying required competencies, developing training pathways and providing training materials are areas that are being looked at and addressed by various groups and projects. In this section, we highlight some of these other projects and activities.
 
+## Number of required RSE graduates
+In order to set up the argument for dedicated RSE education we need to estimate the number of required graduates.
```
Comment (jpthiele) [inline paper_teaching-learning-RSE.md:51]: argument,

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (jpthiele) [inline paper_teaching-learning-RSE.md:53]: 71,733 (otherwise it's hard to compare) researcher**s**

File: paper_teaching-learning-RSE.md [hunk 3]
```diff
@@ -47,6 +47,18 @@ _[Overview of paper sections to add]_
...
+We start from the number of researchers in Germany. The OECD Data [OECD2023] reports a number of 667,394 researchers in 2019.
+In public research alone we have better numbers and there have been 71733 researcher in 2020.
+Taking a rough estimate of requiring around one RSE per 10 researchers (A number that has been put forward in another workshop
+in Paderborn) brings us to a need of around 6.000 RSEs in Germany. Assuming an equal distribution with regard to age
+and an average time in the RSE workforce of 40 years, yields an average need of around 150 RSEs per year.
+If we assume that half of the workforce is made up by researchers coming from other graduates and the other half are properly minted
```
Comment (jpthiele) [inline paper_teaching-learning-RSE.md:57]: How about "from domain specific graduation programs"?

File: paper_teaching-learning-RSE.md [hunk 4]
```diff
@@ -47,6 +47,18 @@ _[Overview of paper sections to add]_
...
+RSE graduates this yields that there is a requirement of around four dedicated RSE education places each producing graduating classes
```
Comment (jpthiele) [inline paper_teaching-learning-RSE.md:58]: graduates, 
"requirement for" instead of "of"?

**Review by jpthiele — APPROVED** [review]
mostly missing punctuation. 
Depending on what public researchers means the calculations seem sound estimates to me.

File: paper_teaching-learning-RSE.md (hunk 2 above)
Comment (jcohen02) [reply paper_teaching-learning-RSE.md:51]: In this context, I think I'd say the comma is not required...

Comment (jcohen02) [reply paper_teaching-learning-RSE.md:51]: In this context, I think I'd say the comma is not required...

File: paper_teaching-learning-RSE.md (hunk 1 above)
Comment (jcohen02) [reply paper_teaching-learning-RSE.md:53]: Suggested edit:
```suggestion
In public research alone we have better numbers and 71,733 researchers were recorded in 2020.
```

File: paper_teaching-learning-RSE.md (hunk 4 above)
Comment (jcohen02) [reply paper_teaching-learning-RSE.md:58]: Also, maybe "suggests" rather than "yields"?

File: paper_teaching-learning-RSE.md [hunk 5]
```diff
@@ -47,6 +47,20 @@ _[Overview of paper sections to add]_
...
+In order to set up the argument for dedicated RSE education we need to estimate the number of required graduates.
+We start from the number of researchers in Germany. The OECD Data [OECD2023] reports a number of 667,394 researchers in 2019.
+In public research alone we have better numbers and there have been 71733 researcher in 2020.
+Taking a rough estimate of requiring around one RSE per 10 researchers (FIXME!! Number anecdotal!!)(A number that has been put forward in another workshop
+in Paderborn) brings us to a need of around 6.000 RSEs in Germany. Assuming an equal distribution with regard to age
+and an average time in the RSE workforce of 40 years, yields an average need of around 150 RSEs per year.
+If we assume that half of the workforce is made up by researchers coming from other graduates and the other half are properly minted
+RSE graduates this yields that there is a requirement of around four dedicated RSE education places each producing graduating classes
+of around 20 persons each year.
+This calculation assumes that RSEs do not move into into industry with their skillset.
```
Comment (jcohen02) [inline paper_teaching-learning-RSE.md:60]:
```suggestion
This calculation assumes that RSEs remain within the research community and do not move into industry with their transferable skillset.
```

**Review by jcohen02 — CHANGES_REQUESTED** [review]
Suggesting a few possible edits/changes. We may want to revisit this as we complete other elements of the paper and see how everything ties together - maybe that will also help us to have a better idea of how realistic this is and offer some more evidence to support the figures?

**Comment by jcohen02** [comment]
From discussion at the meeting on 04-08-2023, we like this content but recognise that the predictions here are likely to have significant inaccuracies because there are a lot of assumptions made.

On this basis, we're not sure that this is necessarily relevant for the initial output focused on competencies but are keen to include this and revisit in due course.

@CaptainSifff, could we deal with the `FIXME` in the content and then merge this?

**Pull request merged**